package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "math-magicians-app",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.16.5",
+    "@testing-library/react": "^13.4.0",
+    "@testing-library/user-event": "^13.5.0",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-scripts": "5.0.1",
+    "web-vitals": "^2.1.4"
+  },
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test",
+    "eject": "react-scripts eject"
+  },
+  "eslintConfig": {
+    "extends": [
+      "react-app"
+    ]
+  },
+  "browserslist": {
+    "production": [
+      ">0.2%",
+      "not dead",
```

Comment:
- [ ] Please make sure that your application runs in the browser without errors so please make sure that you fix this error :point_down:

![image](https://user-images.githubusercontent.com/92303891/218539167-dc3cd304-0c1c-4caf-bad4-d98c394d82a9.png)